source/database-connections/save-mongodb-connection.txt

```diff
@@ -0,0 +1,106 @@
+
+.. _rm-save-mongodb-connection:
+
+============================================
+Save a MongoDB Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To start a migration job, Relational Migrator must connect to your MongoDB
+database using an account with :authrole:`readWrite` access. You can save a new
+MongoDB connection from the Relational Migrator :guilabel:`Connections` page,
+or when creating or modifying a data migration job.
```

Comment:
The procedure below is actually showing a third option, which is to do it from the connections bar. All are valid options, but the procedure and intro text are a bit confusing/inconsistent.

Reply:
Clarified that the standalone topic is for creating the connection from the Connections page. The includes used in the respective pages for creating a new project from a live connection or for creating a data migration job appear on those pages respectively, so the reader doesn't have to navigate away from the larger task.